tests/deployments/test_deployments.py

```diff
@@ -73,5 +73,3 @@ def test_plugin_raising_error():
 def test_target_uri_parsing():
     deployments.get_deploy_client(f_target)
     deployments.get_deploy_client("{target}:/somesuffix".format(target=f_target))
-    with pytest.raises(MlflowException):
```

Comment:
Could we instead test that we can successfully get a deployment client when a double-slash URL is provided? e.g. just include a line like:

```
    deployments.get_deploy_client("{target}://somesuffix".format(target=f_target))	
```